index.py

```diff
@@ -54,7 +59,13 @@ def order(book_id):
     item = library.get_repository('books').find(book_id)
     form = OrderForm(request.form)
     if request.method == 'POST' and form.validate():
-        flash('Thanks for order')
+        load_dotenv()
+        response = requests.get(os.getenv("URI_PROCESSING_ADDRESS"))

```

Comment:
we should pass an order data (first_name, last_name, email, ...) to the processing address. 
you may use 'post' method to implement this feature